recipes/bellmans-gapc/meta.yaml

```diff
@@ -25,9 +23,17 @@ requirements:
     - boost
     - gsl
     - openblas
-source:
-  sha256: a78c39660ebcc7175466c7848f95d33aba277709352a65b6a5c7a2deb5cd10d5
-  url: https://github.com/mmiladi/gapc/archive/git-0.1.tar.gz
+
 test:
+  files:
+    # copy of file in upstream repo: testdata/grammar/nussinov.gap
+    - nussinov.gap
```

Comment:
Can you also remove this, as it is not being used in the tests?

Reply:
good catch, I thought I did remove this already with my commit, but I must have done it wrong. Should be fine by now.